lib/modulesync/renderer.rb

```diff
@@ -10,8 +10,14 @@ def initialize(configs = {})
     end
 
     def self.build(from_erb_template)
```

Comment:
Change this variable name to `template_file` and then you don't need the `File.exist?` check below to work backwards to what was passed in, it's much simpler. Then `from_erb_template = template_file.chomp('erb')`.